evalai/utils/common.py

```diff
@@ -1,4 +1,23 @@
+import click
 from click import echo
+from datetime import datetime
+
+
+class Date(click.ParamType):
+    """
+    Date object parsed using datetime.
+    """
+    name = 'date'
+
+    def __init__(self, format):
+        self.format = format
+
+    def convert(self, value, param, ctx):
+        try:
+            date = datetime.strptime(value, self.format)
+            return date
+        except ValueError:
+            raise self.fail("Incorrect date format, should be {}".format(self.format))
```

Comment:
I think we should change it to `"Incorrect date format, please use {} format".format(self.format)`

Reply:
@RishabhJain2018 Done!